src/sql/src/plan/query.rs

```diff
@@ -73,11 +73,19 @@ pub struct Aug;
 pub struct ResolvedObjectName {
     id: Id,
     raw_name: PartialName,
+    // Whether this object, when printed out, should use [id AS name] syntax. We
```

Comment:
It seemed like @benesch disagreed with me here in Slack, FYI!

Reply:
I guess if we're using `[id AS name]` for everything else, we might as well use it for types? Sorry to point you in the wrong direction!